core/ui/src/components/settings/CloudLogManagerConfigureModal.vue

```diff
@@ -148,6 +154,7 @@ export default {
       toggleEnabled: false,
       address: "",
       tenant: "",
+      cluster_id: "",
```

Comment:
```suggestion
      clusterId: "",
```
Let's use camel case for variables